src/Element_TEST.cc

```diff
@@ -158,11 +158,28 @@ TEST(Element, Include)
 {
   sdf::Element elem;
 
+  SDF_SUPPRESS_DEPRECATED_BEGIN
```

Comment:
BTW Ooh, fancy! Much easier to read than `#pramga RAWRRR` stuff.

Less silly, more concrete example:
https://github.com/RobotLocomotion/drake/blob/v0.27.0/bindings/pydrake/common/test/deprecation_example/cc_module_py.cc#L57-L62

Reply:
For sure! Stole it from DART https://github.com/dartsim/dart/blob/master/dart/common/Deprecated.hpp